### DISCUSSION TITLE
Big Capital accounting

### DISCUSSION THREAD

**Discussion by rkcreation** [coollabsio/coolify#2486] [Service Template Requests]
https://docs.bigcapital.app/deployment/docker

**Comment by iamkevinmccarthy** [comment]
This has been here a while, but here's something:

I've been working in my free time trying to get Bigcapital (@bigcapitalhq) to work on Coolify at all. So far, it's nothing but a headache. I can only get so far: the URL loads, but every attempt to register just hangs. When I check the MySQL logs, this is the bulk of the output stopping this: `[Warning] Access denied for user 'root'@'localhost' (using password: NO)`. Even when I force the MySQL to use `root` instead of the default `bigcapital` database user, it clearly still tries to connect with `root` to no avail.

I'm still trying, but I'm almost afraid if I overcome this hurdle that something else will crop up in its place.

That being said, I would LOVE this to become a Service Template given all the complication baked into the Bigcapital application.
